Resources/ServerInfo/Rules.txt

```diff
@@ -81,7 +81,7 @@ The only exception to the above is theatrical roles such as the clown, boxer, an
     - Certain severe emergencies may make it reasonable for normal crew to defend themselves and their station, such as nuclear operatives, a zombie outbreak, space dragon attack, or other critical situation.
 
 [color=#a4885c]6.[/color] Follow powergaming guidelines:
-If you, in your given role, require an item that does not fall within your job’s usual parameters, have a valid reason to obtain and keep it. Finding something laying around without a clear owner is a valid reason.
+If you, in your given role, require an item that does not fall within your job’s usual parameters, you should have a valid reason to obtain and keep it. Finding something laying around without a clear owner is a valid reason.
```

Comment:
doesnt this mean doctor finding insuls in maint is a valid reason, i.e. only stealing is covered which is basically selfantag

Reply:
Yes, a doctor that finds insulated gloves in maints is not guilty of a rulebreak.

Reply:
this rule was originally for people looting maints roundstart for gamer loot no, thats now explicitly allowed...

Reply:
The way it was enforced in recent history has made rather innocuous activity subject to intrusive bwoinks from admins. Going forward, there isn't really anything wrong with going into maints and finding stuff.

Reply:
> The way it was enforced in recent history has made rather innocuous activity subject to intrusive bwoinks from admins. Going forward, there isn't really anything wrong with going into maints and finding stuff.

So, you're saying, "map more galoshes." Roger 👍😜